HSV:
  Black: [0, 178, 0, 65, 0, 0]
  Blue: [87, 151, 62, 255, 68, 255]
  BlueGoal: [94, 134, 87, 255, 104, 255]
  Red: [162, 13, 91, 255, 88, 255]
  RedGoal: [157, 6, 89, 255, 65, 253]
  White: [0, 180, 0, 92, 255, 255]
  Yellow: [21, 38, 101, 255, 121, 255]
  YellowGoal: [13, 48, 18, 255, 160, 255]
center: [102, 295, 94, 8]
center2: [87, 242, 35, 100]
strategy:
  robot: [0.3, 0.2, 1, 0.025, 0.7, 0.5, 0.1]
  s_planning: [1.5, 0.4, 0.6, 0.2, 30, 10, 0.6, 0.2]
vision_param: [0, 0, 0, 0]
